### PR TITLE
Fix styling bugs

### PR DESCRIPTION
**Pull request by zuzubel**

### CODE REVIEW

File: assets/styles/tailwind.css [hunk 1]
```diff
@@ -24,6 +25,11 @@
   .article-content .ck-editor_demagog-box-wrapper,
   .ck-content .ck-editor_demagog-box-wrapper {
     @apply flex flex-col border-2 border-gray-200 rounded-3xl p-6 lg:px-8 mt-8;
+
+    h3,
+    h2 {
+      margin-top: 0px;
+    }
```
Comment (vaclavbohac) [inline assets/styles/tailwind.css:32]: Does the nesting work here? Didn't know

File: assets/styles/tailwind.css [hunk 2]
```diff
@@ -100,7 +110,7 @@
     @apply underline;
   }
 
-  .article-content figure img {
+  .article-content figure img.avatar {
     @apply !rounded-full;
   }
```
Comment (vaclavbohac) [inline assets/styles/tailwind.css:115]: Do we even need this if it's already defined on the element directly?

image

File: assets/styles/tailwind.css [hunk 3]
```diff
@@ -90,6 +96,10 @@
     @apply !aspect-video !rounded-xl !bg-gray-50 !object-cover;
   }
 
+  .ck-content blockquote {
+    margin-top: 32px;
```
Comment (vaclavbohac) [inline assets/styles/tailwind.css:100]: Can't we use `@apply mt-x` instead?

File: assets/styles/tailwind.css (hunk 1 above)
Comment (zuzubel) [reply assets/styles/tailwind.css:32]: Yes it does.

**Pull request merged**